packages/wxt/src/__tests__/storage.test.ts

```diff
@@ -1,6 +1,6 @@
 import { fakeBrowser } from '@webext-core/fake-browser';
 import { describe, it, expect, beforeEach, vi, expectTypeOf } from 'vitest';
-import { browser } from '../browser';
+import { browser } from 'wxt/browser';
```

Comment:
Because we're using `mkdist` now, we can import from `wxt/browser` and it will be resolved from the NPM module instead of using the local file, and this will be replaced with `wxt/browser/chrome` when using `extensionApi: "chrome"`